src/target/TargetAttribution.php

```diff
@@ -24,6 +24,7 @@ public function attributes()
             'name'      => StringAttribute::class,
             'label'     => StringAttribute::class,
             'remoteid'  => StringAttribute::class,
+            'state'     => StringAttribute::class,
```

Comment:
How does this change relate to HP-1275?

AFAIS the target does not contain a State attribute.